values/prometheus-operator/prometheus-operator.gotmpl

```diff
@@ -102,6 +102,26 @@ prometheus:
     {{- $domain := printf "team-%s.%s" $teamId $v.cluster.domainSuffix }}
     {{- tpl (readFile "../../helmfile.d/snippets/blackbox-targets.gotmpl") (dict "teamId" $teamId "services" $teamServices "domain" $domain) | nindent 6 }}
   {{- end }}
+{{- end }}
+{{- if $p.remoteWrite.enabled }}
+    remoteWrite:
+    - url: "{{ $p.remoteWrite.rwConfig.target }}"
+      {{- if $p.remoteWrite.rwConfig.basicAuth.enabled }}
+      basicAuth:
+        username:
+          name: prometheus-remote-write-basic-auth
+          key: username
+        password:
+          name: prometheus-remote-write-basic-auth
+          key: password
+      {{- end }}
+      tlsConfig:
+        insecureSkipVerify: true
+      {{- if $p.remoteWrite.otomiThanos }}
+      headers:
```

Comment:
why the `cluster` header is needed for Thanos?

Reply:
To label data coming from a specific cluster